frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1519,6 +1541,27 @@ impl<T: Config> Pallet<T> {
 
 		// Kill snapshots.
 		Self::kill_snapshot();
+
+		// Reset election block counter.
+		ElectionBlocksCount::<T>::kill();
+	}
+
+	// Returns `true` if the emergency phase should be throttled.
```

Comment:
```suggestion
	/// Returns `true` if instead of going into emergency phase, we should prolong/throttle the existing election round.
```

1. note the `///` instead of `//`